data/pizza.json

```diff
@@ -1148,7 +1136,7 @@
 			"ingredients": [
 				"isbergssallad", "tomat", "gurka", "feferoni", "lök", "kyckling", "pommes frites eller ris"
```

Comment:
[This too should include kebabsås](https://sannegarden.com/meny/johanneberg/#:~:text=L%C3%B6k%2C%20Feferoni%2C%20Kebabs%C3%A5s%20Mild%2C%20Kyckling%2C%20Gurka%2C%20Isbergssallad%2C%20Pommes%20Frites%2C%20Tomater)